Makefile

```diff
@@ -54,6 +61,34 @@ build-image-ocp-example: build-base-image ## Build a rag-content container image
 build-base-image: ## Build base container image
 	podman build -t $(TORCH_GROUP)-road-core-base -f Containerfile.base --build-arg FLAVOR=$(TORCH_GROUP)
 
+start-postgres: ## Start postgresql from the pgvector container image
+	podman run -d --name pgvector --rm -e POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) \
+	 -p $(POSTGRES_PORT):5432 \
+	 -v $(PWD)/postgresql/data:/var/lib/postgresql/data:Z pgvector/pgvector:pg16
+
+start-postgres-debug: ## Start postgresql from the pgvector container image with debugging enabled
+	mkdir -pv ./postgresql/data ./output
```

Comment:
Should we be making these also for `start-postgres` ?

Reply:
Updated `Makefile` as suggested.